File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
      if(x<0) return false;
        long long int n=x;
      
        long long int rev_x=0;
        
        while(n){
             int rem = n%10;
             rev_x = rev_x*10 + rem;
             n/=10;
        }
        
     return x == rev_x;   
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x<0:
            return False
        n = x
        rev_x=0
        
        while n>=1:
            rem = n%10
            rev_x = rev_x*10 + rem
            n=n//10
        
        return rev_x ==x